### PR TITLE
Issue 242: Check that -Project was not specified, not that is null.

### PR DESCRIPTION
**Pull request by ILMTitan**
Check that a parameter with a config default was not specified. Previously, we had check that the parameter was not null.

Also fix a problem with writing the progress of random operations.

### CODE REVIEW

**Comment by ILMTitan** [comment]
Fixes issues #242  and #284.

File: Google.PowerShell/Compute/GceCmdlet.cs [hunk 1]
```diff
@@ -132,7 +132,7 @@ private void WriteWarnings(Operation op)
         private void WriteProgress(Operation op, string progressMessage)
         {
 
-            int activityId = op.Id.GetHashCode();
+            int activityId = Math.Abs(op.Id.GetHashCode());
```
Comment (iantalarico) [inline Google.PowerShell/Compute/GceCmdlet.cs]: This seems odd to me.  Is this common practice?  If not a comment would be nice (and below)

**Comment by iantalarico** [comment]
LGTM

File: Google.PowerShell/Compute/GceCmdlet.cs [hunk 2]
```diff
@@ -131,8 +131,8 @@ private void WriteWarnings(Operation op)
         ///  method will generate a message from the operation.</param>
         private void WriteProgress(Operation op, string progressMessage)
         {
-
-            int activityId = op.Id.GetHashCode();
+            // ProgressRecord was throwing an error on negative activityIds.
```
Comment (chrsmith) [inline Google.PowerShell/Compute/GceCmdlet.cs]: "was throwing" => "will throw"

**Pull request merged**